src/planscape/planscape/settings.py

```diff
@@ -364,3 +367,19 @@
 OUTPUT_DIR = BASE_DIR / "output"
 
 DEFAULT_EST_COST_PER_ACRE = config("DEFAULT_EST_COST_PER_ACRE", 2470, cast=float)
+
+
+# SINGLE QUEUE BEHAVIOR
+USE_CELERY_FOR_FORSYS = config("USE_CELERY_FOR_FORSYS", False, cast=bool)
+
+# CELERY
+CELERY_BROKER_URL = config("CELERY_BROKER_URL", "redis://localhost:6379/0")
```

Comment:
nit: Should we put these into a CELERY namespace?

Reply:
I was originally thinking it could just be grouped under CELERY { } for organization.

But on second thought, since some of this is going to be set from ENV, that would potentially just be more confusing, because we'd want to keep the CELERY_ prefix on any ENV vars. 
And then it's more confusing to translate between CELERY_SOME_SETTING and SOME_SETTING in settings.py

So apologies, I take back my suggestion, let's keep it as it is.